docs/terms_and_conditions.md

```diff
@@ -0,0 +1,148 @@
+1. Usługodawca oraz Użytkownik i Użytkownik niezalogowany zobowiązani są do naprawienia szkody, jaką druga strona poniosła na skutek niewykonania lub nienależytego wykonania przez nich obowiązków wynikających z Regulaminu, chyba że ich niewykonanie lub nienależyte wykonanie było następstwem okoliczności, za które nie ponoszą odpowiedzialności, w szczególności na skutek działania siły wyższej.
+2. Usługodawca, który otrzymał urzędowe zawiadomienie lub wiarygodną wiadomość o bezprawnym charakterze danych wykorzystywanych przez Użytkownika i Użytkownika niezalogowanego w ramach korzystania z Serwisu i uniemożliwił dostęp do takich danych lub informacji, nie ponosi względem nich odpowiedzialności za powstałą szkodę.
+3. Usługodawca nie ponosi wobec Użytkownika i Użytkownika niezalogowanego naruszających Regulamin odpowiedzialności za jakiekolwiek szkody powstałe na skutek zaprzestania świadczenia im Usług, w tym na skutek usunięcia Konta.
+4. Usługodawca nie ponosi ponadto odpowiedzialności za:
+a. jakiekolwiek szkody wyrządzone osobom trzecim, powstałe w wyniku korzystania przez Użytkownika i Użytkownika niezalogowanego z Usług w sposób sprzeczny z Regulaminem lub przepisami prawa,
+b. treści udostępniane przez Użytkownika i Użytkownika niezalogowanego w wyniku korzystania z Usług, które to treści naruszają prawo lub chronione prawem dobra osób trzecich,
+c. informacje oraz materiały pobrane, zamieszczone w Serwisach lub wysyłane za pośrednictwem sieci Internet przez Użytkownika i Użytkownika niezalogowanego,
+d. utratę przez Użytkownika i Użytkownika niezalogowanego danych spowodowaną działaniem czynników zewnętrznych (np. awaria sprzętu) lub też innymi okolicznościami niezależnymi od Usługodawcy (działanie osób trzecich), w tym zawinionymi przez Użytkownika i Użytkownika niezalogowanego,
+e. szkody wynikłe na skutek braku ciągłości dostarczania Usług, będące następstwem okoliczności, za które Usługodawca nie ponosi odpowiedzialności (siła wyższa, działania i zaniechania osób trzecich itp.),
+f. podania przez Użytkownika i Użytkownika niezalogowanego nieprawdziwych, nieaktualnych lub niepełnych danych lub informacji,
+g. nieprzestrzeganie przez Użytkownika i Użytkownika niezalogowanego postanowień Regulaminu.
+
+**VIII. OCHRONA DANYCH OSOBOWYCH** (w Polityce Prywatności???)
+1. Każdy Użytkownik oraz Użytkownik niezalogowany ma prawo do ochrony jego prywatności przez Usługodawcę.
+2. Zasady dotyczące przetwarzania przez Usługodawcę danych osobowych Użytkowników oraz Użytkowników niezalogowanych określa Polityka Prywatności.
+
+**IX. POSTĘPOWANIE REKLAMACYJNE** (???)
+1. Użytkownik i Użytkownik niezalogowany mają prawo składać reklamacje w sprawach dotyczących realizacji Usług.
+2. Reklamacje rozpatruje Usługodawca.
+3. Prawidłowo złożona reklamacja powinna zawierać co najmniej następujące dane:
+a. oznaczenie Użytkownika lub Użytkownika niezalogowanego (imię, nazwisko, adres poczty elektronicznej),
+b. adres Serwisu, którego reklamacja dotyczy,
```

Comment:
To chyba niepotrzebne - nie mamy więcej niż jednego serwisu w ramach volontulo :)